src/htmlParser.js

```diff
@@ -0,0 +1,391 @@
+// A stateless HTML Parser written in JavaScript
+// Based on http://ejohn.org/blog/pure-javascript-html-parser/
+// Turns an HTML string into a token iterator.
+
+//TODO(#39)
+/*globals console:false*/
+var supports = (function() {
+  var target = {};
+
+  var html;
+  var work = this.document.createElement('div');
```

Comment:
I've attempted to include this fork in my webpack project, and i'm running issues on this line. 
I'm assuming the intent here is for the window object to be passed in as the this context. however I don't see this done anywhere